tests/integration/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Odd i didn't write this and have no idea where it came from?  These should not even exist these look like the old tests.